### ISSUE TITLE
ModuleNotFoundError: No module named 'fp'

### ISSUE THREAD

**Issue by Undertone0809** [VinciGit00/Scrapegraph-ai#95]
**Describe the bug**

I run https://github.com/VinciGit00/Scrapegraph-ai/blob/main/examples/openai/smart_scraper_openai.py and show error: ModuleNotFoundError: No module named 'fp'

**To Reproduce**
I run the following example and get error

```python
"""
Basic example of scraping pipeline using SmartScraper
"""

import os
from dotenv import load_dotenv
from scrapegraphai.graphs import SmartScraperGraph
from scrapegraphai.utils import prettify_exec_info

load_dotenv()


# ************************************************
# Define the configuration for the graph
# ************************************************

openai_key = os.getenv("OPENAI_APIKEY")

graph_config = {
    "llm": {
        "api_key": openai_key,
        "model": "gpt-3.5-turbo",
    },
}

# ************************************************
# Create the SmartScraperGraph instance and run it
# ************************************************

smart_scraper_graph = SmartScraperGraph(
    prompt="List me all the projects with their description.",
    # also accepts a string with the already downloaded HTML code
    source="https://perinim.github.io/projects/",
    config=graph_config
)

result = smart_scraper_graph.run()
print(result)

# ************************************************
# Get graph execution info
# ************************************************

graph_exec_info = smart_scraper_graph.get_execution_info()
print(prettify_exec_info(graph_exec_info))
```

```text

D:\Application\Conda\envs\agentkit\python.exe D:\Projects\githubhub\agentkit\tmp.py 
Traceback (most recent call last):
  File "D:\Projects\githubhub\agentkit\tmp.py", line 7, in <module>
    from scrapegraphai.graphs import SmartScraperGraph
  File "D:\Application\Conda\envs\agentkit\lib\site-packages\scrapegraphai\graphs\__init__.py", line 5, in <module>
    from .smart_scraper_graph import SmartScraperGraph
  File "D:\Application\Conda\envs\agentkit\lib\site-packages\scrapegraphai\graphs\smart_scraper_graph.py", line 5, in <module>
    from ..nodes import (
  File "D:\Application\Conda\envs\agentkit\lib\site-packages\scrapegraphai\nodes\__init__.py", line 5, in <module>
    from .fetch_node import FetchNode
  File "D:\Application\Conda\envs\agentkit\lib\site-packages\scrapegraphai\nodes\fetch_node.py", line 9, in <module>
    from ..utils.remover import remover
  File "D:\Application\Conda\envs\agentkit\lib\site-packages\scrapegraphai\utils\__init__.py", line 8, in <module>
    from .proxy_rotation import proxy_generator
  File "D:\Application\Conda\envs\agentkit\lib\site-packages\scrapegraphai\utils\proxy_rotation.py", line 4, in <module>
    from fp.fp import FreeProxy
ModuleNotFoundError: No module named 'fp'
```

**Comment by PeriniM** [comment]
hey for now install the missing module with `pip install free-proxy`.
We are going to fix it by adding it to the dependencies

**Comment by PeriniM** [comment]
Fixed it in 7f1c3b7
thank you for pointing it out!
